bin/utils/subkey/Cargo.toml

```diff
@@ -7,7 +7,6 @@ license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
 homepage = "https://substrate.io"
 repository = "https://github.com/paritytech/substrate/"
 readme = "README.md"
-publish = false
```

Comment:
Wont it need a description to publish?

Reply:
Oh right yeah thanks. I'll add a description in a bit.